tests/e2e/svmotion_detached_volume.go

```diff
@@ -435,10 +448,883 @@ var _ = ginkgo.Describe("[csi-block-vanilla] [csi-block-vanilla-parallelized] Re
 		e2eVSphere.verifyDatastoreMatch(volumeID, []string{destDsUrl})
 
 		ginkgo.By("Verify that the relocated CNS volumes are compliant and have correct policy id")
-		storagePolicyExists, err = e2eVSphere.VerifySpbmPolicyOfVolume(volumeID, policyName)
+		storagePolicyMatches, err = e2eVSphere.VerifySpbmPolicyOfVolume(volumeID, policyName)
 		e2eVSphere.verifyVolumeCompliance(volumeID, true)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
-		gomega.Expect(storagePolicyExists).To(gomega.BeTrue(), "storage policy verification failed")
+		gomega.Expect(storagePolicyMatches).To(gomega.BeTrue(), "storage policy verification failed")
+
+	})
+
+	/*
+		Start attached volume's relocation and then expand it
```

Comment:
the file name suggests that we have tests only for detached volumes here, either change the filename or move the attached volume tests to a new file

Reply:
Renamed the file